arangod/V8Server/v8-analyzers.cpp

```diff
@@ -320,6 +320,10 @@ void JS_Create(v8::FunctionCallbackInfo<v8::Value> const& args) {
       TRI_V8_THROW_TYPE_ERROR("<properties> must be an object");
     }
   }
+  // properties(if exists) at the end should be parsed into object 
+  if(properties.use_count() && !properties->slice().isObject()) {
```

Comment:
I dislike the usage of `use_count()` here a bit.
Wouldn't it be more intuitive to write just
```suggestion
  if(properties && !properties->slice().isObject()) {
```

Reply:
Agreed, Moreover, it was decided to slightly rewrite the code to completely get rid of shared_ptr. Will update shortly.